src/app/lib/config/toggles/index.js

```diff
@@ -41,6 +41,20 @@ const toggles = {
       enabled: false,
     },
   },
+  // stage: {
+  //   chartbeatAnalytics: {
+  //     enabled: false,
+  //   },
+  //   mediaPlayer: {
+  //     enabled: false,
+  //   },
+  //   mpulse: {
+  //     enabled: true,
+  //   },
+  //   navOnArticles: {
+  //     enabled: false,
+  //   },
+  // },
```

Comment:
Why bother adding commented-out code?

Reply:
That sthe config for the stage environment. It trips some snapshot tests and will be needed for the stage env tests which are coming with B/G. This is just a pre-requisite PR I will clean most of this code up once we get the full B/G pipeline to work.